src/style_corect.css

```diff
@@ -0,0 +1,86 @@
+/* @font-face {
+  font-family: Avenir;
+  src: url('fonts/Avenir-Book.ttf') format('truetype');
+  font-weight: normal;
+  font-style: normal;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url('fonts/Avenir-Heavy.ttf') format('truetype');
+  font-weight: bold;
+  font-style: normal;
+}
+
+body {
+  font-family: Avenir, Arial, Helvetica, sans-serif;
+} */
+
+.search-bar__input {
+  /* display: block;
+  width: 100%; */
+  margin-top: 20px;
+}
+
+.search-bar__input--big {
+  /* height: 70px; */
```

Comment:
The 'height: 70px;' property is commented out. Verify if this is intentional, as it might affect the layout of the '.search-bar__input--big' class.